### DISCUSSION TITLE
How to build and run the examples?

### DISCUSSION THREAD

**Discussion by styxlab** [Josh-McFarlin/remix-image#26] [Q&A] [answered]
I am having some trouble building and executing the examples. If I change into the example directory and run the usual `yarn && yarn build` I get a bunch of errors, for example:

```
✘ [ERROR] [plugin server-bare-modules] Package subpath './package.json' is not defined by "exports" in /home/cws/remix-img/examples/basic/node_modules/remix-image/package.json

    node_modules/@remix-run/dev/compiler/plugins/serverBareModulesPlugin.js:125:32:
      125 │   let packageJsonFile = require.resolve(`${packageName}/package.j...
          ╵                                 ^
```

What's the right way to build and execute the examples?

**Comment by Josh-McFarlin** [comment, marked as answer]
Hi Joost,

Apologies for the late response. Are you still running into this issue? I have two thoughts

1. Make sure the `postinstall` script is being run for Remix, they customize the package for each installation environment, which can lead to some errors

2. Unfortunately Remix has changed their examples/environment configurations, so the examples are kinda outdated. Before I got busy, I started working on the next major update of this package and I updated the examples in the process. [You can see them here.](https://github.com/Josh-McFarlin/remix-image/tree/workspace/examples) However, I want to note that I did not have the time to finish this update, and there were several bugs in progress that were preventing Remix Image from working. So feel free to try the updated examples linked, but you should make sure you are using the NPM published package instead of building the broken version from that branch.

**Comment by Josh-McFarlin** [comment]
@styxlab Today I released v1.0.0 of Remix-Image, and I have updated all the examples using `npx create-remix` so they are all up to date and run correctly!
